json-endpoints/class.wpcom-json-api-update-post-v1-2-endpoint.php

```diff
@@ -13,6 +13,11 @@ function write_post( $path, $blog_id, $post_id ) {
 			add_action( 'rest_api_inserted_post', array( $GLOBALS['publicize_ui']->publicize, 'async_publicize_post' ) );
 		}
 
+		// 'future' is an alias for 'publish' for now
+		if ( 'future' === $input['status'] ) {
```

Comment:
When autosaving, this throws a notice for `undefined variable input`.  Think it just needs an `isset`